botocore/useragent.py

```diff
@@ -60,6 +60,15 @@
     'S3_TRANSFER': 'G',
     'ENDPOINT_OVERRIDE': 'N',
     'SIGV4A_SIGNING': 'S',
+    'FLEXIBLE_CHECKSUMS_REQ_CRC32': 'U',
+    'FLEXIBLE_CHECKSUMS_REQ_CRC32C': 'V',
+    'FLEXIBLE_CHECKSUMS_REQ_CRC64': 'W',
```

Comment:
I'm thinking we should just refer to this as `FLEXIBLE_CHECKSUMS_REQ_CRC64NVME` instead of `FLEXIBLE_CHECKSUMS_REQ_CRC64`. At the end of the day, all that matters is that we register the `'W'` feature id. It's kinda weird to use`CRC64` and then special case for `CRC64NVME`.

What do you think?